central/graphql/resolvers/vulnerability_counter.go

```diff
@@ -104,6 +104,25 @@ func mapVulnsToVulnerabilityCounter(vulns []*storage.EmbeddedVulnerability) *Vul
 	return counter
 }
 
+func mapImageCVEsToVulnerabilityCounter(fixable, unFixable []*storage.ImageCVE) *VulnerabilityCounterResolver {
```

Comment:
Implement an interface so that we don't have to implement such func for each vuln type.
Something like,
```
type CVEWithSev interface {
GetId()
GetSeverity()
}
```